website/docs/docs/on-premises/releases/2.1.0.md

```diff
@@ -0,0 +1,48 @@
+---
+title: 2.1.0
+---
+
+## Release date
+
+10 July 2025
+
+## Features
+
+- Added supports and verify sensitive information to be obfuscated when sending to 
```

Comment:
Is OTel officially supported in this release yet? Or are these just commits that relate to it?

If we include it, we should link it to the OTel docs (PR incoming for that).

Reply:
> Is OTel officially supported in this release yet? Or are these just commits that relate to it?

The code included [this changes](https://github.com/pactflow/pactflow-application/pull/2477/files). Probably @impurist can confirm

Reply:
Thanks. Good to merge once we confirm and tidy up (I'll mark as approved so you can make the updates and merge).